packages/backend/src/services/searchService/integrationTests/utilities/logTags.js

```diff
@@ -0,0 +1,17 @@
+/* eslint-disable import/no-extraneous-dependencies */
```

Comment:
Think we should separate dev dependencies and non dev dependencies. Instead of disabling the rule we can adjust it like "import/no-extraneous-dependencies": ["error", {"devDependencies": ["**/*.test.js", "**/*.spec.js"]}]